### ISSUE TITLE
Allow empty bill description

### ISSUE THREAD

**Issue by whtv** [eneiluj/cospend-nc#28]
Since there are categories, a description is often redundant.

**Comment by eneiluj** [comment]
Hi, right, bill title could be optional. It is now :grin:

It would be nice if you (or anyone else) could try it to see if there is any side effect. It's available in v1.3.9-2-nightly (which is only compatible with NC 22).
You can easily get it with an occ command (adapt to your context):
```
sudo -u www-data php ./occ app:update --allow-unstable cospend
```
To get back to stable release, just get to the app settings, disable, delete and "download and install" the app.

**Comment by eneiluj** [comment]
Feel free to reopen if it does not properly works.
